diff --git a/Tweak.x b/Tweak.x
index b0a600c..839d863 100644
--- a/Tweak.x
+++ b/Tweak.x
@@ -4,12 +4,12 @@
 
 #import "../YTVideoOverlay/Header.h"
 #import "../YTVideoOverlay/Init.x"
-#import "../YouTubeHeader/YTColor.h"
-#import "../YouTubeHeader/YTMainAppVideoPlayerOverlayViewController.h"
-#import "../YouTubeHeader/YTMainAppVideoPlayerOverlayView.h"
-#import "../YouTubeHeader/YTMainAppControlsOverlayView.h"
-#import "../YouTubeHeader/YTPlayerViewController.h"
-#import "../YouTubeHeader/QTMIcon.h"
+#import <YouTubeHeader/YTColor.h>
+#import <YouTubeHeader/YTMainAppVideoPlayerOverlayViewController.h>
+#import <YouTubeHeader/YTMainAppVideoPlayerOverlayView.h>
+#import <YouTubeHeader/YTMainAppControlsOverlayView.h>
+#import <YouTubeHeader/YTPlayerViewController.h>
+#import <YouTubeHeader/QTMIcon.h>
 
 #define TweakKey @"YouLoop"
 #define IS_ENABLED(k) [[NSUserDefaults standardUserDefaults] boolForKey:k]
